Nginx-Certbot.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Configure Nginx and TLS Certifications on Your Server
+layout: topic
+categories: guide
+---
+
+Now that you have a domain and a web server, you need to configure 
+the server to serve content to your domain.
+
+The two most common web server software suites are 
+[`nginx`](https://nginx.org/) and [`apache`](https://www.apache.org/). 
+This wiki page will cover web server configuration with `nginx` 
+because it is easier to configure for first-time users. The wiki page 
+assumes that the machine used to serve content is a Digital Ocean 
+droplet, but the specific steps won't differ widely between services.
+
+## Step 1: Install `nginx`
+
+Installation is relatively easy.
+- ssh into your droplet.
+- To confirm that the software is not currently installed, run the 
+  following command:<br>
+  ```
+  $ which nginx
+  $
+  ```
+  <br>
+  If you see no executable path before the prompt reappears, then you 
+  will need to install `nginx`.
+- Install `nginx` with the following command:<br>
+  ```
+  $ sudo apt-get install nginx
+  ```
+  <br>
+  When prompted with `Do you want to continue? [Y/n]`, type yes and 
+  press return.
+- To confirm that `nginx` was installed, rerun `which nginx`.
+  You should now see a path to the executable in the console.
+
+## Step 2: Get Your Website Code onto the Server
+
+Assuming that your code is in a `git` repository, clone the repository 
+onto your machine.
+
+In our example, we will be serving 
+[this GitHub repository](https://github.com/UWB-ACM/website-example-2) 
+at [phry.me](http://phry.me).
+
+To clone the repository, run the following commands and replace the 
+GitHub URL with the URL to your own repository.
+
+```
+$ cd /var/www
```

Comment:
Might be helpful to mention that nginx assumes website files are under `/var/www/`

Reply:
Addressed in 7c1a9d051f9c55fa86e76013b274a40a644cea45